aplt/runner.py

```diff
@@ -77,24 +91,25 @@ def connect(self, processor):
         self._connect_waiters.append(processor)
         connectWS(self._factory, contextFactory=self._factory_context)
 
-    def send_notification(self, processor, url, data, ttl):
+    def send_notification(self, processor, url, data, ttl, claims=None):
         """Send out a notification to a url for a processor"""
         url = url.encode("utf-8")
+        headers = {"TTL": str(ttl)}
+        if claims is None and self._claims:
+            claims = self._claims
+        if self._vapid and claims:
+            headers.update(self._vapid.sign(claims))
```

Comment:
signing takes approx 0.026s on my box. Is that within the threshold for adding yields?